spec/02-integration/02-cmd/14-vault_spec.lua

```diff
@@ -0,0 +1,111 @@
+local helpers = require "spec.helpers"
+
+
+describe("kong vault", function()
+  lazy_setup(function()
+    helpers.get_db_utils(nil, {}) -- runs migrations
+  end)
+
+  after_each(function()
+    helpers.kill_all()
+  end)
+
+  lazy_teardown(function()
+    helpers.clean_prefix()
+  end)
+
+  it("vault help", function()
+    local ok, stderr, stdout = helpers.kong_exec("vault --help")
+    assert.matches("Usage: kong vault COMMAND [OPTIONS]", stderr, nil, true)
```

Comment:
Removed the options from the command
```suggestion
    assert.matches("Usage: kong vault COMMAND", stderr, nil, true)
```

Reply:
There are default options, see above.